src/api/spec/features/webui/maintenance_workflow_spec.rb

```diff
@@ -34,8 +34,6 @@
     visit package_show_path(project: update_project, package: package)
 
     click_link('Branch package')
-    # we need this find to wait for the dialog to appear
-    expect(find(:id, 'branch_dialog')).to have_text('Do you really want to branch package')
```

Comment:
True... I just read `We need this to wait[...]` - I didn't read the `find` in it. You're right. I just need to remove the find...